packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec

```diff
@@ -14,9 +14,9 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
```

Comment:
Perhaps the spec file should be updated to allow for automatic updating?

Reply:
sorry, not sure I follow, do you mean the template spec file?

Reply:
These 2 lines allow it:

https://github.com/theforeman/foreman-packaging/blob/7cdee5d764539d5342560c3b4631dd216d6aa31f/gem2rpm/scl.spec.erb#L32
https://github.com/theforeman/foreman-packaging/blob/7cdee5d764539d5342560c3b4631dd216d6aa31f/gem2rpm/scl.spec.erb#L62

It also needs to know which template to use to generate the content:
https://github.com/theforeman/foreman-packaging/blob/7cdee5d764539d5342560c3b4631dd216d6aa31f/gem2rpm/scl.spec.erb#L1

Then in the future if you use `bump_rpm.sh` it'll automatically bump the dependencies. I started the conversion of a lot of the spec files, but didn't complete it.

Reply:
Ah okay, I'll go ahead and update it.